.github/workflows/update-upstream-provider.yml

```diff
@@ -0,0 +1,96 @@
+env:
```

Comment:
@stack72 This would not normally be a part of the PR - it's because I haven't merged the latest changes to master.